docs/setup/compose_based.md

```diff
@@ -0,0 +1,61 @@
+---
+title: Compose Based Deployment
+---
+
+<head>
+  <title>Environment Setup</title>
+  <meta
+    name="description"
+    content="Here we'll deploy our pipeline in the Docker Container"
+  />
+</head>
+
+Here we'll deploy our pipeline in the Docker Container using compose.
+
+## Pre-requisites:
```

Comment:
These pre-requisites are wrong. we only need docker-compose, docker and make for this. Not jenkins vault etc.